cli/flox/doc/flox-containerize.md

```diff
@@ -4,23 +4,61 @@ section: 1
 header: "Flox User Manuals"
 ...
 
+```{.include}
+./include/experimental-warning.md
+```
 
 # NAME
 
 flox-containerize - export an environment as a container image
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] containerize [ `<options>` ]
+```
+flox [ `<general-options>` ] containerize
+     [-d=<path> | -r=<owner/name>]
+     [-o=<path>]
+```
 
 # DESCRIPTION
 
 Export a Flox environment as a container image.
-The image is dumped to stdout and should be piped to `docker load`.
+The image is written to `<path>`.
+Use `docker load` to load the image into docker.
+
+Running the container will behave like running `flox activate`
```

Comment:
Suggestion: I don't think this will be clear to a user. Could you be more explicit?